packages/teams-js/src/private/files.ts

```diff
@@ -296,6 +296,70 @@ export namespace files {
     extension: string;
   }
 
+  export enum CloudStorageProviderFileAction {
```

Comment:
exported enums should have comments and be marked as @hidden since I believe they will show up in the documentation